server/configuration.go

```diff
@@ -73,6 +73,12 @@ type clientConfig struct {
 	MaxRecordingDuration *int
 }
 
+const (
+	defaultRecDurationMinutes = 60
+	minRecDurationMinutes     = 15
+	maxRecDurationMinutes     = 180
```

Comment:
@justinegeffen I am temporarily setting these as min/max limits. We can discuss tomorrow in our call if they need tuning and then can be added to the documentation.

Reply:
Brilliant, that works for me. Thanks.
(i missed the meeting as I was in surgery, sorry!)

Reply:
No worries, hope everything went well :)